bundles/org.openhab.binding.juicenet/src/main/resources/OH-INF/i18n/juicenet.properties

```diff
@@ -0,0 +1,21 @@
+# FIXME: please substitute the xx with a proper locale, ie. de
+# FIXME: please do not add the file to the repo if you add or change no content
+# binding
+binding.juicenet.name = JuiceNet Binding
+binding.juicenet.description = This binding interfaces to a JuiceNet EV Charger. 
+
+# thing types
+thing-type.juicenet.sample.label = <Your localized Thing label>
+thing-type.juicenet.sample.description = <Your localized Thing description>
+
+# thing type config description
+thing-type.config.juicenet.sample.hostname.label = <Your localized config parameter label>
+thing-type.config.juicenet.sample.hostname.description = <Your localized config parameter description>
+thing-type.config.juicenet.sample.password.label = <Your localized config parameter label>
+thing-type.config.juicenet.sample.password.description = <Your localized config parameter description>
+thing-type.config.juicenet.sample.refreshInterval.label = <Your localized config parameter label>
+thing-type.config.juicenet.sample.refreshInterval.description = <Your localized config parameter description>
+
+# channel types
+channel-type.juicenet.sample-channel.label = <Your localized Channel label>
+channel-type.juicenet.sample-channel.description = <Your localized Channel description>
```

Comment:
Please remove all examples and instructions, and regenerate file: https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file